pkg/cloud/common/resources.go

```diff
@@ -53,3 +57,42 @@ func getPDB(config config.OperatorConfig) (*policyv1.PodDisruptionBudget, error)
 		},
 	}, nil
 }
+
+// getService returns a common service for the cloud-controller-manager on port 10258,
+// for a given platform.
+func getService(config config.OperatorConfig) *corev1.Service {
+	matchLabels := map[string]string{
+		CloudControllerManagerProviderLabel: config.GetPlatformNameString(),
+	}
+	name := fmt.Sprintf("%s-cloud-controller-manager", strings.ToLower(config.GetPlatformNameString()))
+
+	return &corev1.Service{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Service",
+			APIVersion: "core/v1",
```

Comment:
```suggestion
			APIVersion: "v1",
```

Not sure what we use to create this object, but setting typemeta should maybe not really be required?!

Reply:
Yeah, this was to make existing unit tests happy :( We have to do the same for the PDB above, so it's matching the existing pattern. 

I agree though, It's ugly :(